hd/etc/home.txt

```diff
@@ -27,7 +27,7 @@
       <i class="fa fa-search fa-fw fa-xs" aria-hidden="true"></i>%nn;
       <span class="sr-only">[*search/case sensitive]0</span>%nn;
     </a>
-    <a role="button" class="btn btn-sm btn-link p-0 border-0" href="%url_setx.p.n.oc.i;&i=%apply;get_random(base.real_nb_persons.v)"
+    <a role="button" class="btn btn-sm btn-link p-0 border-0" href="%url_setx.p.n.oc.i;&i=%apply;get_random(nb_of_persons.v)"
```

Comment:
Now it is implemented on all page even if the page isn't a function of an individual, we should change this hyperlink in home to : %if i/pnoc is in url we stay on the tool (url_setx), else we pick a random ind. (%prefix;&i=…) %end; !